### DISCUSSION TITLE
Does not work with .js for nuxt.config?

### DISCUSSION THREAD

**Discussion by johnkomarnicki** [nuxt/content#1086] [General]
When converting my nuxt.config to .js, it throws this error.

![Screen Shot 2022-05-15 at 1 38 45 PM](https://user-images.githubusercontent.com/52672963/168486344-9d97157b-8528-4433-bd52-9581341e44b9.png)
 
Is anyone else having this issue?

**Comment by nozomuikuta** [comment]
@johnkomarnicki 

Could you provide a repository link so that other people can debug.
